tests/app/clients/test_zendesk_sell.py

```diff
@@ -1,9 +1,12 @@
+import json
```

Comment:
Can you add a test for `send_create_service`?

Reply:
test cases added